OasisDEX/oasis-borrow
# Lw/7770 new headline

# [New Vault headline](https://app.shortcut.com/oazo-apps/story/7770/implement-new-version-of-vault-headline)


## Changes 👷‍♀️
- Implemented new headline according to design https://www.figma.com/file/XeMo7s0ZJgXgCHmOd588QY/Discover-Oasis-%235366?node-id=2904%3A16028  
  
## How to test 🧪
- Check vault headlines in various vaults in various states see if everything fits, no layout is broken, here are some examples:

![image](https://user-images.githubusercontent.com/6871923/215821479-1737e7c8-d230-421e-a782-de317bdd094d.png)

![image](https://user-images.githubusercontent.com/6871923/215821531-a820fcea-df12-4066-bb6c-23d261d30ab3.png)

![image](https://user-images.githubusercontent.com/6871923/215821593-49956454-25fe-4add-906f-35ff6c7b7166.png)

![image](https://user-images.githubusercontent.com/6871923/215821645-68547474-9bf8-4d9c-8f7e-b920cff7bbbf.png)
This branch is started from dev so it doesn't yet have validations, but I  just can simulate edge caser that was found by @piotrkonowrocki 
![image](https://user-images.githubusercontent.com/6871923/215821719-362fce0e-4e62-4df6-969f-c44e7547ee88.png)

Let's say hypotheticaly in one of languages we support message will be even longer still looks fine

![image](https://user-images.githubusercontent.com/6871923/215821907-a861afd7-a48a-44ff-aa34-420821523156.png)
![image](https://user-images.githubusercontent.com/6871923/215822291-7eff1891-2b5b-404b-b6c1-9e930c9d499c.png)

AAVE
![image](https://user-images.githubusercontent.com/6871923/215822371-074714b6-4333-451a-a82f-c40d1c172465.png)
